documentation/modules/overview/con-overview-mirrormaker2.adoc

```diff
@@ -14,21 +16,27 @@ active/passive cluster configuration:: An active/passive configuration has an ac
 The expectation is that producers and consumers connect to active clusters only.
 A MirrorMaker 2 cluster is required at each target destination.
 
+NOTE: In most deployments, write operations should be made only to local topics. 
```

Comment:
In this file we refer to `local` and `remote` instead of `source` and `target` again. It's defined lower down but not at this point, so we either need to define it earlier or switch the wording